CONTRIBUTION.md

```diff
@@ -0,0 +1,161 @@
+   * [Documentation Structure](#documentation-structure)
+      * [Using newdoc to create templates for new documents](#using-newdoc-to-create-templates-for-new-documents)
+   * [Writing documentation](#writing-documentation)
+      * [Style](#style)
+      * [Using vale to check the style](#using-vale-to-check-the-style)
+      * [Adding Images](#adding-images)
+   * [Building Documentation](#building-documentation)
+   * [Publishing documentation](#publishing-documentation)
+* [Getting Support](#getting-support)
+
+
+## Code of Conduct of Eclipse Che documentation project
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+### Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional setting
+
+### Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at [http://contributor-covenant.org/version/1/4][version]
+
+[homepage]: http://contributor-covenant.org
+[version]: http://contributor-covenant.org/version/1/4/
+
+## Pull Request Process 
+
+1. Eclipse Che documentation project follows forking workflow. To learn more, read [Atlassian Git Tutorial, Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow). Before you start working, fork the repository, make sure you create all branches in your own fork.
+2. `master`  is the default working branch. Unless your change is intended for a specific version, send the PR against a `master`. See the Release branches to learn more.
+3. Specify as much information as possible in the PR template. 
+
+### Release branches
+[WIP]
+
+## Creating documentation
+
+### AsciiDoc Formatting
+Eclipse Che documentation use AsciiDoc for formatting. See [AsciiDoc Writer's Guide](https://asciidoctor.org/docs/asciidoc-writers-guide/) for syntax and general help with AsciiDoc.
```

Comment:
Technically, I'd say AsciiDoc is used for markup, not formatting. (Also, "uses" instead of "use" here.)